core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -775,6 +776,32 @@ class GroupMetadataManagerTest {
   }
 
   @Test
+  def testloadGroupWithStaticMember() {
```

Comment:
Do we have unit test coverage on compatibility? I.e. old formatted data can be loaded with new versioned byte code with new fields set to default (null) values?

Reply:
@guozhangwang mind giving me an example for compatibility test? I look around and haven't found one good example.

Reply:
You can for example take a look at this PR: https://github.com/apache/kafka/pull/6528

When we update the consumer protocol, we added unit test to make sure old versioned code can still deser it, and similarly in this case, we need to change that new versioned code can still deser old versioned data.

Reply:
Thanks!